nuxeo-core/nuxeo-core-event/src/main/java/org/nuxeo/ecm/core/work/WorkComputation.java

```diff
@@ -84,11 +84,13 @@ public void signalStop() {
     public void processRecord(ComputationContext context, String inputStreamName, Record record) {
         work = deserialize(record.getData());
         try {
-            if (work.isIdempotent() && workIds.contains(work.getId())) {
+            if (work.isCoalescing() && WorkStateHelper.getLastOffset(work.getId()) > context.getLastOffset()) {
+                log.warn("Coalescing work id with new instance scheduled: " + work.getId() + ", skipping");
```

Comment:
"Coalescing work with the same id: <id> scheduled, skipping" if this is expected because of rendition triggers it is going to be very frequent and log level should be info.

Reply:
I used the same log level as idempotent works already executed, should I lower it too ?

Reply:
Yes please. INFO level should be about things of interest to people administering the server (start/stop/important events)